openedx/core/djangoapps/schedules/migrations/0008_add_new_start_date_fields.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
```

Comment:
Nit: this filename should be `0008_add_new_start_date_field.py` (field is singular)

Reply:
got it.